spec/requests/pages/conditions_controller_spec.rb

```diff
@@ -314,10 +310,6 @@
         it "redirects to the confirm exit page deletion page" do
           expect(response).to redirect_to confirm_change_exit_page_path(form_id: form.id, page_id: page.id, condition_id: condition.id, params: { answer_value: "Wales", goto_page_id: 3 })
         end
-
-        it "does not call save! for the condition" do
-          expect(ConditionRepository).not_to have_received(:save!)
-        end
```

Comment:
Can we have a test that checks that the condition is not updated to replace this one?

Reply:
Added this, also added a missing check that the condition was updated for the happy path